backend/main.py

```
@@ -3,465 +3,483 @@
 from flask_cors import CORS
 from flask_socketio import SocketIO
-from flask_migrate import Migrate, upgrade
 
 # Add project paths
-sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..'))
-sys.path.append(os.path.join(os.path.dirname(__file__), '..', '..', 'shared'))
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
+sys.path.append(os.path.join(os.path.dirname(__file__), "..", "..", "shared"))
 
 # Import utilities - using relative imports
 from .utils.error_handler import SwarmError, create_error_response, handle_errors
```

Flake8 (7.2.0)

[error] module level import not at top of file

(E402)